doc/classes/EditorUndoRedoManager.xml

```diff
@@ -0,0 +1,118 @@
+				If this is the first operation, the [param object] will be used to deduce target undo history.
+			</description>
+		</method>
+		<method name="add_undo_property">
+			<return type="void" />
+			<param index="0" name="object" type="Object" />
+			<param index="1" name="property" type="StringName" />
+			<param index="2" name="value" type="Variant" />
+			<description>
+				Register a property value change for "undo".
+				If this is the first operation, the [param object] will be used to deduce target undo history.
+			</description>
+		</method>
+		<method name="add_undo_reference">
+			<return type="void" />
+			<param index="0" name="object" type="Object" />
+			<description>
+				Register a reference for "undo" that will be erased if the "undo" history is lost. This is useful mostly for nodes removed with the "do" call (not the "undo" call!).
+			</description>
+		</method>
+		<method name="commit_action">
+			<return type="void" />
+			<param index="0" name="execute" type="bool" default="true" />
+			<description>
+				Commit the action. If [param execute] is true (default), all "do" methods/properties are called/set when this function is called.
+			</description>
+		</method>
+		<method name="create_action">
+			<return type="void" />
+			<param index="0" name="name" type="String" />
+			<param index="1" name="merge_mode" type="int" enum="UndoRedo.MergeMode" default="0" />
+			<param index="2" name="custom_context" type="Object" default="null" />
+			<description>
+				Create a new action. After this is called, do all your calls to [method add_do_method], [method add_undo_method], [method add_do_property], and [method add_undo_property], then commit the action with [method commit_action].
+				The way actions are merged is dictated by the [param merge_mode] argument. See [enum UndoRedo.MergeMode] for details.
+				If [param custom_context] object is provided, it will be used for deducing target history (instead of using the first operation).
+			</description>
+		</method>
+		<method name="get_history_undo_redo" qualifiers="const">
+			<return type="UndoRedo" />
+			<param index="0" name="id" type="int" />
+			<description>
+				Returns the [UndoRedo] object associated with the given history [param id].
+				[param id] above [code]0[/code] are mapped to the opened scene tabs (but it doesn't match their order). [param id] of [code]0[/code] or lower have special meaning (see [enum SpecialHistory]).
+				Best used with [method get_object_history_id]. This method is only provided in case you need some more advanced methods of [UndoRedo] (but keep in mind that directly operating on the [UndoRedo] object might affect editor's stability).
+			</description>
+		</method>
+		<method name="get_object_history_id" qualifiers="const">
+			<return type="int" />
+			<param index="0" name="object" type="Object" />
+			<description>
+				Returns the history id deduced from the given [param object]. It can be used with [method get_history_undo_redo].
```

Comment:
That's a matter for a different PR, but I think we should harmonize our usage of "ID" and "id" in the classref.

```
$ rg -g'*.xml' " ID " | wc -l
122
$ rg -g'*.xml' " id " | wc -l
25
```